layers/+source-control/git/README.org

```diff
@@ -136,6 +155,37 @@ To enable it you have to add the following lines to your
 #+END_SRC
 
 ** Forge
+
+Magit Forge can view and create issues & pull requests with forges
+(e.g. GitHub, GitLab)
+
+Magit Forge requires a GitHub or GitLab user name and will prompt for a value if not
+defined in  =~/.gitconfig=
+Or define your forge identity using the following command in a terminal window
+
+#+begin_src shell
+  git config --global github.name "username"
+#+end_src
+
+For GitLab, use gitlab.name instead of github.name
```

Comment:
Use code brackets for `gitlab.name` instead to differentiate it from clear text.